README.md

```diff
@@ -130,6 +130,7 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [Angel List](https://angel.co/jobs) - Startup Jobs. Search by going to Job Type, and selecting "Remote OK".
   1. [Authentic Jobs](https://authenticjobs.com/#remote=true) – Full-time and freelance job opportunities for web, design, and creative professionals.
   1. [Codestunts](https://codestunts.com)
+  1. [Remote Crypto Jobs](https://cryptojobslist.com/remote) - #1 job board to find and post crypto, bitcoin and blockchain jobs.

```

Comment:
Why is this called Remote Crypto Jobs, and not Crypto Jobs List? :)

Reply:
Hey!
Mmm, idk 😅 at the time, I felt like it'll sound more relevant to your list, and was not particular about my site's name.
But feel free to rename to Crypto Jobs List.
Or let me know — I'll make a separate pull request.

Reply:
Could you just update this pull request by changing the name back? That way it will be alphabetised again. Other links have the remote option specified; we know that it'll be for remote jobs by dint of being on this list.